src/views/CreateTransferOrder.vue

```diff
@@ -268,13 +268,28 @@ watch(queryString, (value) => {
 }, { deep: true });
 
 onIonViewWillEnter(async () => {
+  emitter.on('clearScannedOrderItem', clearScannedOrderItem as any);
   emitter.emit('presentLoader');
   await fetchTransferOrderDetail(route?.params?.orderId as string);
   await fetchProductInformation();
   await store.dispatch('util/fetchFacilities', getProductStoreId())
   emitter.emit('dismissLoader');
 });
 
+const clearScannedOrderItem = (removedProductId: string) => {
```

Comment:
```suggestion
const clearSearchedProduct = () => {
```

No need to pass productId as this will only be called when we want to clear the searched result